### PR TITLE
open nodePort on host with port 10050/10051

### PR DESCRIPTION
**Pull request by tgruenert**
#### What this PR does / why we need it:
NodePort for services open Port different from original Zabbix ports.

#### Which issue this PR fixes
Open ports on node via NodePort open Ports >32000 (if you using rancher kubernetes). To use the original Zabbix ports 10051 and 10052 the solution is to open this port directly on the host where it is deployed.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 name: zabbix
-version: 0.2.1
+version: 0.2.2
```
Comment (aeciopires) [inline Chart.yaml:4]: Hi @tgruenert!

In **develop**  branch the version of chart is 0.3.0. Can you change here to 0.3.1 or major?

File: .circleci/config.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ jobs:
       - run:
           name: helm-github-pages
           environment:
-            - GITHUB_PAGES_REPO: cetic/helm-charts
+            - GITHUB_PAGES_REPO: evermind/helm-charts
```
Comment (aeciopires) [inline .circleci/config.yml:11]: Hi @tgruenert!

Can you change here to **cetic/helm-charts** repo?

**Comment by aeciopires** [comment]
Hi @tgruenert !

Thank you very much for your collaboration. I made some comments that need to be adjusted in your Pull Request.

In **CONTRIBUTING.md** file has instructions for install helm-docs command to update documentation and update table with new values.

```
... Install the follow packages: git, kubectl, helm, helm-docs. See this tutorial...

and

...Run command helm-docs to update content of README.md file using the README.md.gotmpl template...
```

Can you execute these instructions?

File: .circleci/config.yml (hunk 2 above)
Comment (tgruenert) [reply .circleci/config.yml]: done

File: Chart.yaml (hunk 1 above)
Comment (tgruenert) [reply Chart.yaml]: done

**Comment by tgruenert** [comment]
@aeciopires thank you for your tips. i changed things as you requested. hope it helps to merge. but i can´t handle the build-fail.

**Review by alexnuttinck — APPROVED** [review]

**Comment by alexnuttinck** [comment]
Thanks for your PR @tgruenert, I merge it now. I fixed the CI.

**Pull request merged**